src/Sarif.Multitool.Library/Rules/RuleResources.resx

```diff
@@ -379,7 +379,7 @@ Semantics: Assuming the reader of the log file (an end user or another tool) has
     <value>Each result location must provide the property 'physicalLocation.artifactLocation.uri'. GitHub Advanced Security code scanning will not display a result whose location does not provide the URI of the artifact that contains the result.</value>
   </data>
   <data name="GH1001_ProvideRequiredLocationProperties_Error_MissingLocationProperty_Text" xml:space="preserve">
-    <value>{0}: '{1}' is absent. GitHub Advanced Security code scanning will not display a result location that does not provide the URI of the artifact that contains the result.</value>
+    <value>{0}: '{2}' is absent. GitHub Advanced Security code scanning will not display a result location that does not provide the URI of the artifact that contains the result.</value>
```

Comment:
Is changing here desired

Reply:
Yes, {1} will always be the service name, e.g. GHAS or
 ADO AS